go/test/endtoend/vtorc/general/vtorc_test.go

```diff
@@ -286,3 +288,68 @@ func TestCircularReplication(t *testing.T) {
 	// check that the writes still succeed
 	utils.VerifyWritesSucceed(t, clusterInfo, primary, []*cluster.Vttablet{replica}, 10*time.Second)
 }
+
+// TestSemiSync tests that semi-sync is setup correctly by vtorc if it is incorrectly set
+func TestSemiSync(t *testing.T) {
+	// stop any vtorc instance running due to a previous test.
+	utils.StopVtorc(t, clusterInfo)
```

Comment:
I'm surprised you need this. Should a previous `vtorc` not be torn down at the end of the previous test?

Reply:
The previous tests actually don't cleanup their `vtorc` instances. All the tests are responsible for removing any previous instance of vtorc, vttablets and cleaning MySQL. This test is using a new clusterInstance so it has to remove any previously running vtorc